Automattic/wp-calypso
# Anchor: Add podcast title step UI

#### Changes proposed in this Pull Request

* Add podcast title step UI, leaning into the updated framework styles rather than the old styles.
* I don't understand why the framework header (WP logo in the upper left corner) isn't part of the markup in this flow; it should be.... 🤔 

**Visual**
<img width="977" alt="Screen Shot 2022-04-13 at 9 30 59 AM" src="https://user-images.githubusercontent.com/2124984/163191666-2218b925-aa04-495c-9357-00eb95ed1e6d.png">


#### Testing instructions

* Switch to this PR, navigate to `/setup/?flags=signup/anchor-fm&siteSlug=[yoursite.wordpress.com]`
* You should see the above as the title step. Clicking Continue should progress to the design selection step.
* There is no actual saving functionality going on here yet; it's just the UI.

Related to #62666
